### ISSUE TITLE
About Kitti data

### ISSUE THREAD

**Issue by yuehua-Song666** [koutilya-pnvr/SharinGAN#6]
Hi,
I'm struggling with the Kitti data u used in ur project, can u plz tell me which item did u download from the official Kitti website?
Maybe the data u used is Kitti 360? But I didn't find data which is labeled as 2011.
Thank you so much for ur help.

**Comment by yuehua-Song666** [comment]
I think I figure it out.
Sorry for disturbing u.
Thank you and stay safe.

**Comment by koutilya-pnvr** [comment]
No problem @yuehua-Song666. Let me know in case you have any other questions. Stay safe.

**Comment by yuehua-Song666** [comment]
Hi,
Sorry for disturbing u again, I have prepared all the datasets which are needed in ur project, but when I ran the code, I got some problem, below is the error message:

AssertionError: Depth does not exist. 
Depth path: Depth_baseline_all_syn/Bilinear_model_pseudo_labels/2011_09_26/2011_09_26_drive_0015_sync/image_02/data/0000000026.png

I'm not sure what is this file (Depth_baseline_all_syn/Bilinear_model_pseudo_labels) for, I didn't have this file. what should I prepare?

I tried to figure out what happened, but I failed. Sorry about that, below is what I did:
When I  clicked line 100 in Solver.py, I jumped to Kitti_dataloader_training.py line 26 and I don't understand line 56. In line 56, u used data_info[0], is that the depth information we need? Or we should use .bin files in Kitti. 

Thank you in advance

Yuehua Song
